compiler/luci/import/src/GraphBuilder.cpp

```diff
@@ -53,6 +55,17 @@ void GraphBuilder::build(const circle::OperatorT &op, GraphBuilderContext *conte
 
   CircleNode *node = build_node(op, input_nodes, context->graph());
 
+  for (uint32_t i = 0; i < operators.size(); ++i)
+  {
+    const circle::OperatorT &ref_op = *operators[i];
+    if (&ref_op == &op)
+    {
+      node->annot(std::make_unique<luci::CircleNodeID>(i));
```

Comment:
I guess node ID will be overwritten on export?

Reply:
In ONE, yes it will be.
Backend may be different with ONE :)

Reply:
@cgbahk To be specific, once operators are imported(get annotation), it won't be overwritten. We will just create new operator list when exports to be optimized(or sth) model.